### DISCUSSION TITLE
Schedules completely ignored

### DISCUSSION THREAD

**Discussion by xjru** [borgbase/vorta#1859] [FAQ] [answered]
Hi,

what could be the reason that Vorta completely ignores all my schedules? I set up two schedules yesterday (for two different repositories) using "backup daily". Today, nothing happened at the set times while the computer was turned on and Vorta was running. There is one "scheduled create" entry in the log many hours before the scheduled time but no corresponding archive was created.

I just tried to add a test file and set a new schedule in case Vorta doesn't do anything if nothing has changed in the file system. But again, nothing happened. I can start backups manually and they work just fine, creating new archives and log entries. It's only the scheduling part that doesn't work at all.

I'm using Vorta 0.8.11 with borg 1.2.6 on macOS 14.1.1

Thanks

**Comment by m3nu** [comment, marked as answer]
We can't help with old versions. You should try the latest 0.9 beta for macos.

**Comment by real-yfprojects** [comment]
The logs might be interesting too. In generally opening an issue when having an issue is the best approach. That's where we include a template/form so that we have a good starting point.

**Reply by xjru** [reply]
I didn't think there was an issue with the software. I thought I might be doing something wrong as no one else had reported any scheduling problems.

Anyway, I tried the latest development version and the issue is indeed fixed as m3nu suggested. 

One thing I noticed while trying to get to the bottom of this is that daily backups are always scheduled for the next day even when the time is in the future. It's not a problem at all, but it initially thwarted my attempts to find out what was going on.

Sorry for posting in the wrong place.

**Reply by real-yfprojects** [reply]
> One thing I noticed while trying to get to the bottom of this is that daily backups are always scheduled for the next day even when the time is in the future. It's not a problem at all, but it initially thwarted my attempts to find out what was going on.

This shouldn't be the case.

https://github.com/borgbase/vorta/blob/567a3546ae09878cf64f9f10c5c6f7bb7254720e/src/vorta/scheduler.py#L315-L326
